src/transformers/modeling_tf_longformer.py

```diff
@@ -992,7 +1004,12 @@ def call(
 
         embedding_output = self.embeddings(input_ids, position_ids, token_type_ids, inputs_embeds, training=training)
         encoder_outputs = self.encoder(
-            [embedding_output, extended_attention_mask, output_attentions, output_hidden_states, padding_len],
+            embedding_output,
+            attention_mask=extended_attention_mask,
+            padding_len=padding_len,
+            output_attentions=output_attentions,
+            output_hidden_states=output_hidden_states,
+            return_dict=return_dict,
```

Comment:
@patrickvonplaten After the previous fix was done, there was an issue about `output_attentions` being a `tf.Tensor` that couldn't be used as a Python bool. The fix was to pass the arguments to the encoder as kwargs instead of as a dict.

Reply:
I see, that's were good to know, thanks!

Reply:
It would be good to try the same in TF T5 which also passes the inputs as a a big list. Maybe this was the cause of our bugs.